### PR TITLE
Hi-1 from dev1_change1

### PR DESCRIPTION
**Pull request by isac-fernandesdev1**
Simple change to logic: Hi-1 from dev1_change1

### CODE REVIEW

File: HelloABC.html [hunk 1]
```diff
@@ -29,5 +29,6 @@
 Hello-1 from server branch
 Hello-2 from server branch
 Hello-3 from server branch
+Hi-1 from dev1_change1
```
Comment (isac-fernandes) [inline HelloABC.html:32]: Please add Hi-2 to the logic.

**Comment by isac-fernandes** [comment]
Hi-2 needs to be added.

File: HelloABC.html [hunk 2]
```diff
@@ -29,5 +29,7 @@
...
+Hi-2 from dev1_change1
```
Comment (isac-fernandes) [inline HelloABC.html:33]: Please add Hi-3 to the logic.

**Review by isac-fernandes — CHANGES_REQUESTED** [review]
Hi-3 needs to be added

File: HelloABC.html (hunk 2 above)
Comment (isac-fernandesdev1) [reply HelloABC.html:33]: Please check, Hi-3 added

**Comment by isac-fernandesdev1** [comment]
Hi-3 just added, please review.

**Review by isac-fernandes — APPROVED** [review]

**Pull request merged**